TypeCobol.Analysis.Test/DfaCallPgmReport.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Compiler;
+using System.IO;
+using TypeCobol.Analysis.Report;
+
+using static TypeCobol.Analysis.Test.CfgTestUtils;
+
+namespace TypeCobol.Analysis.Test
+{
+    /// <summary>
+    /// Dfa test for call module report
+    /// </summary>
+    [TestClass]
+    public class DfaCallPgmReport
```

Comment:
From #1751.
Would be nicer if we had a way to factorize test data/results with associated CLI test.

Reply:
@mayanje, simply add data and results files as links in CLI.Test, links will point to physical files in TypeCobol.Analysis.Test

Reply:
@fm-117 I not sure that links cross projects is a good idea.

Reply:
Yeah I know but it's acceptable for test projects, we had link for skeleton file at some point so I guess it's ok.

Reply:
It seems that we no longer use link to skeleton file cross projects. So I keep on thinking that link to test files cross project, is not good even for test projects.

Reply:
Actually the idea comes from @smedilol and I think it's appropriate

Reply:
The problem is we have more and more tests. 
If there are changes in DFA, we'll need to review a lot of tests.
In previous version of TypeCobol we has link between project TypeCobol and TypeCobol.Grammar, so I don't see any problems to do that again.